drivers/usb/dwc3/core.h

```diff
@@ -1110,7 +1110,9 @@ struct dwc3_scratchpad_array {
  *	3	- Reserved
  * @dis_metastability_quirk: set to disable metastability quirk.
  * @dis_split_quirk: set to disable split boundary.
+ * * @has_dsm_for_softreset: set if we want to use BIOS to do core soft reset
  * @has_dsm_for_softreset: set if we want to use BIOS to do core soft reset
```

Comment:
Minor nitpick but you've ended up with 2 copies of the comment for "@has_dsm_softreset" after the conflict resolution.

Also it should probably go after the comment for the new "suspend" bit flag that got added (i.e. the comments seem to match the order of the fields in the struct.